src/TriggerBinding/SqlTriggerListener.cs

```diff
@@ -31,6 +31,7 @@ internal sealed class SqlTriggerListener<T> : IListener, IScaleMonitor<SqlTrigge
         private const int ListenerStarted = 2;
         private const int ListenerStopping = 3;
         private const int ListenerStopped = 4;
+        public const int DefaultMaxChangesPerWorker = 1000;
```

Comment:
I had added the below code comment to address review suggestion. Can you add it back to top of `DefaultMaxChangesPerWorker` definition?

```
            // NOTE: please ensure the Readme file and other public documentation are also updated if this value ever
            // needs to be changed.
```